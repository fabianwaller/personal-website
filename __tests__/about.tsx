import About from "@/app/about/page";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("About", () => {
  it("renders about page unchanged", () => {
    const component = render(<About />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
