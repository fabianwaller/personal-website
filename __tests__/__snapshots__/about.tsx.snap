// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders about page unchanged 1`] = `
<DocumentFragment>
  <section
    class="w-full py-8"
    id="about"
  >
    <div
      class="container relative ml-auto mr-auto w-screen max-w-custom px-8 md:px-0 mb-12"
    >
      <div
        class="text-center"
      >
        <h2>
          About me
        </h2>
        <span
          class="block text-sm"
        >
          What I'm Doing Now
        </span>
      </div>
    </div>
    <div
      class="container relative ml-auto mr-auto w-screen max-w-custom px-8 md:px-0"
    >
      <div
        class="grid items-start gap-6 md:grid-cols-2"
      >
        <div
          class="flex flex-wrap items-center justify-start gap-2"
        >
          <span
            class="inline-block divide-border rounded-lg border-2 border-solid px-4 py-2 text-sm text-title-normal"
          >
            computer science student
          </span>
          <span
            class="inline-block divide-border rounded-lg border-2 border-solid px-4 py-2 text-sm text-title-normal"
          >
            fullstack web development
          </span>
          <span
            class="inline-block divide-border rounded-lg border-2 border-solid px-4 py-2 text-sm text-title-normal"
          >
            mobile development
          </span>
        </div>
        <div>
          <p>
            I'm currently studying computer science at Saarland University in Germany and I share my knowledge and projects on this website. I'm mainly interested in fullstack web and mobile development, software engineering and artificial intelligence. I also enjoy playing football, travelling around the world and reading books in my spare time.
          </p>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
